Airline_Web_API/Airline_Web_API/Controllers/FlightsController.cs

```diff
@@ -28,18 +30,33 @@ public async Task<ActionResult<IEnumerable<Flight>>> GetFlights([FromQuery] int
                 .Include(flight => flight.DepartureAirport)
                 .Include(flight => flight.ArrivalAirport)
                 .Include(flight => flight.Tickets)
+                    .ThenInclude(ticket => ticket.TicketType)
                 .AsQueryable();
+
             flights = flights.Where(flight =>
                 flight.DepartureAirportId == departureAirportId
                 && flight.ArrivalAirportId == arrivalAirportId
                 && flight.DepartureDate.Date == date.Date
                 && flight.Tickets
                     .Count(ticket => ticket.TicketsLeftNumber >= ticketsNumber) > 0
             );
-                
-            return await flights
-                .OrderBy(flight => flight.Id)
-                .ToListAsync();
+
+            var flightsSearchResults = await flights.OrderBy(flight => flight.Id).ToListAsync();
+
+            var config = new MapperConfiguration(config =>
+            {
+                config.CreateMap<Airport, AirportViewModel>();
+                config.CreateMap<TicketType, TicketTypeViewModel>();
+                config.CreateMap<Ticket, TicketsViewModel>();
+                config.CreateMap<Flight, FlightViewModel>();
+            }
+           );
+
+            var mapper = new Mapper(config);
```

Comment:
As this does not change between requests you can move it to static constructor and use static mapper in code.